src/main/java/Ivan/Encapsulation1/Employee.java

```diff
@@ -0,0 +1,37 @@
+package Ivan.Encapsulation1;
+
+public class Employee {
+    private int employeeID;
+    private String name;
+    private double salary;
+
+    public Employee(int employeeID, String name, double salary) {
```

Comment:
why do you need constructor with parameters which you ignore and don't initialize ?